LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Ekino PHP Team
```

Comment:
Ekino seems sufficient, or we need to update the header in php-cs for consistency